CRM/Banking/PluginImpl/Matcher/DefaultOptions.php

```diff
@@ -44,6 +44,7 @@ function __construct($config_name) {
     if (!isset($config->default_financial_type_id)) $config->default_financial_type_id = 1;
     if (!isset($config->createnew_value_propagation)) $config->createnew_value_propagation = array();
     if (!isset($config->manual_default_financial_type_id)) $config->manual_default_financial_type_id = NULL;
+    if (!isset($config->preserve_receive_date)) $config->preserve_receive_date = 0; // Set to 1 to preserve the receive_date (instead of overwriting it with the booking_date). See issue #409.
```

Comment:
I see this follows existing formatting, but conditions (`if`) without brackets (`{…}`) should really be avoided. Also, the comment should be put above the condition.

Reply:
```suggestion
    // Set to 1/TRUE to preserve the receive_date (instead of overwriting it with the booking_date). See issue #409.
    $config->preserve_receive_date = (bool) ($config->preserve_receive_date ?? FALSE);
```

Reply:
Thanks! I agree with the changes, even though I think it doesn't necessarily contribute to the readability of the code to suddenly deviate from the scheme in the nth repeated line.